CMakeLists.txt

```diff
@@ -23,21 +23,27 @@ project(OpenVINOGenAI
         HOMEPAGE_URL "https://github.com/openvinotoolkit/openvino.genai"
         LANGUAGES CXX)
 
-option(ENABLE_CONTINUOUS_BATCHING "" OFF)
-option(ENABLE_APPS "Enable C++ continuous batching apps. Ignored if ENABLE_CONTINUOUS_BATCHING is OFF" ON)
-
 # Find OpenVINODeveloperPackage first to compile with SDL flags
 find_package(OpenVINODeveloperPackage QUIET
              PATHS "${OpenVINO_DIR}")
 if(NOT OpenVINODeveloperPackage_FOUND)
     find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 endif()
 
+# check that SDPA to PA transformtion exists
+get_target_property(ov_include_dirs openvino::runtime INTERFACE_INCLUDE_DIRECTORIES)
+find_file(spda_to_pa_header sdpa_to_paged_attention.hpp
+    PATHS ${ov_include_dirs}
+    PATH_SUFFIXES openvino/pass
+    DOC "Path to sdpa_to_paged_attention.hpp header"
+    NO_CACHE REQUIRED NO_DEFAULT_PATH)
```

Comment:
this code is not required

it was a temporary solution to ensure that OV version to SPDA -> PA transformation is used. Now, we can just set minimal required version to find_package